symphony-bdk-examples/bdk-app-spring-boot-example/src/main/java/com/symphony/bdk/examples/app/spring/ExtensionAppApplication.java

```diff
@@ -2,12 +2,26 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 @SpringBootApplication
 public class ExtensionAppApplication {
 
   public static void main(String[] args) {
-    SpringApplication.run(ExtensionAppApplication.class, args);
+    SpringApplication application = new SpringApplication(ExtensionAppApplication.class);
+    application.setAdditionalProfiles("ssl");
+    application.run(args);
   }
 
+  @Bean
+  public WebMvcConfigurer corsConfigurer() {
+    return new WebMvcConfigurer() {
+      @Override
+      public void addCorsMappings(CorsRegistry registry) {
+        registry.addMapping("/**").allowedOrigins("*");
```

Comment:
Maybe we should add a comment saying this is not a recommended production setup

Reply:
Yes I will update the documentation how to make it work as well

Reply:
We need something to configure cors in `app-starter`, following https://github.com/SymphonyPlatformSolutions/symphony-bdk-bot-sdk-java#cors